src/pages/ChatPage/components/ChatInput.tsx

```
@@ -20,14 +111,61 @@ type FormValues = {
+    mediaRecorder.onstop = () => {
+      const audioBlob = new Blob(audioChunks.current, {
+        type: "audio/webm",
+      });
+
+      const url = URL.createObjectURL(audioBlob);
+      setAudioUrl(url);
+
+      console.log("Voice message blob:", audioBlob);
+
+      // Stop mic access
+      stream.getTracks().forEach((track) => track.stop());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)